ui/docs/manual/deploying/anonymous-role.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Anonymous Role
+---
+
+# Anonymous Role
+
+A single scope `assume:anonymous` will be granted for all calls (with or without credentials).
+The `anonymous` role is aimed at private Taskcluster installations. With this, it will
+be possible to prevent any useful access to the Taskcluster deployment without credentials
+allowing for greater administrator control over "public" endpoints.
+
+## Public Deployments
+
+All API calls are granted the scope `assume:anonymous`.
+In order to allow unauthenticated API, grant the relevant scopes to the `anonymous` role.
+A "public" deployment of Taskcluster, allowing unauthenticated read-only access, would grant the following scopes:
+
+  - `auth:current-scopes`
```

Comment:
Now that I think about it, this list will grow/shrink over time. How can we make sure we keep it up-to-date? Should we be generating this list somehow?

Reply:
At the moment there isn't anything automated to keep it up to date but it's definitely something we should think about. I'll go ahead and create an issue for that assuming it's fine to do this in a follow up.

Reply:
Filed https://github.com/taskcluster/taskcluster/issues/3737.